source/atlas-search.txt

```diff
@@ -0,0 +1,21 @@
+.. _java-atlas-search:
+
+============
+Atlas Search
+============
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: full text, text analyzer, meta, pipeline, scoring, Lucene
+   :description: Learn about how to use Atlas Search in the {+driver-long+}.
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+See :atlas:`Atlas Search </atlas-search/>` in the MongoDB Atlas Manual.
```

Comment:
S: We usually just use "manual" for server, and "documentation" for atlas (applies to all)
```suggestion
See :atlas:`Atlas Search </atlas-search/>` in the MongoDB Atlas documentation.
```